test/feeds.test.js

```diff
@@ -369,6 +370,20 @@ describe('test DELETE /feeds/:id endpoint', () => {
   });
 });
 
+it('should return 404 when id is less than 10 characters', async () => {
+  const id = '123456789';
+  const res = await request(app).delete(`/feeds/${id}`);
+
+  expect(res.status).toEqual(404);
```

Comment:
This should be 400, indicating format is wrong vs. 404, indicating the format is right but the resource is not found.